packages/terra-table/src/subcomponents/Section.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SectionHeader from './SectionHeader';
+
+const propTypes = {
+  /**
+   * The children list items passed to the component.
+   */
+  children: PropTypes.node,
+  /**
+   * The numberOfColumns to be used as a descriptor for assistive technology.
+   */
+  numberOfColumns: PropTypes.number.isRequired,
```

Comment:
Looks like there's some extra props defined here.

Reply:
The extras are being passed through custom props. We are only unpacking the props relevant to collapsing here.